playwright-e2e/pages/singular/enrollment/pre-screening-page.ts

```diff
@@ -82,12 +82,14 @@ export default class PreScreeningPage {
       hasHeartDefect = true
     } = opts;
 
-    await this.age().fill(age);
+    await this.age().fill(age, { waitRequestAfter: false });
     await this.country().select().selectOption(country);
```

Comment:
I am surprised you need it here as the UI is designed to disable button while PATCH is in flight. But You do need it, right?

Reply:
Are you asking why we need to wait for triggered pending request to finish after type/select/click?

An example. Say, we have a button called 'Next', when clicked, it navigates to a new page. This button will be disabled during PATCH (sent a request & wait for response) request. However, the problem is, if targeted field is not a **Required** field, button was already enabled before sending PATCH request. 🤷‍♂️
Playwright playback speed is very fast. Playwright checks the state of button after type/select/click on a field and clicks immdiately because button was enabled (It doesn't check again and doesn't know any pending PATCH requests). Triggered request sometimes is a second or more late after field loses focus. We have to handle this special synchronization ourself.

Let me know if your question was not about this.